pymc3/distributions/continuous.py

```diff
@@ -1017,7 +1035,22 @@ def logp(self, value):
 
 
 class Bound(object):
-    """Creates a new bounded distribution"""
+    R"""
+    Creates a new upper, lower or upper+lower bounded distribution
+
+    Parameters
+    ----------
+    distribution : pymc3 distribution
+        Distribution to be transformed into a bounded distribution
+    lower : float (optional)
+        Lower bound of the distribution
+    upper : float (optional)
+
+    Example
+    -------
+    boundedNormal = pymc3.Bound(pymc3.Normal, lower=0.0)
+    par = boundedNormal(mu=0.0, sd=1.0, testval=1.0)
+    """
```

Comment:
Looks great. Mainly I thought that we should document the behavior of `transform='infer'` though.